commons-datastore/commons-datastore-core/pom.xml

```diff
@@ -22,19 +26,11 @@
             <groupId>org.apache.commons</groupId>
             <artifactId>commons-lang3</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.slf4j</groupId>
-            <artifactId>slf4j-api</artifactId>
-        </dependency>
         <dependency>
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
This `slf4j-simple` dependency is needed for basic logging during testing (thats why it has the scope). I would not remove it.